modules/javafx.web/src/main/native/Source/WebCore/platform/graphics/java/ComplexTextControllerJava.cpp

```diff
@@ -92,6 +92,10 @@ unsigned jGetEnd(jobject jRun)
 
 unsigned jGetCharOffset(jobject jRun, unsigned glyphIndex)
 {
+    if (!jGetGlyphCount(jRun)) {
+        return { };
```

Comment:
Shouldn't this return an explicit (perhaps an out-of-band) value?

Reply:
Sure, I think it is better to return the same value as `TextRun.getCharOffset`. i.e it just returns the `glyphIndex`.